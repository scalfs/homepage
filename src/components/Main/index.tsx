const Main = ({
  title = 'Vitor Escalfoni',
  description = 'Software Engineer specialized in mobile app development.'
}) => (
  <main className="bg-[#06092b]">
    <h1>{title}</h1>
    <h2>{description}</h2>
  </main>
)

export default Main
