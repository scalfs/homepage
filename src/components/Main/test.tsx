import { render } from '@testing-library/react'

import Main from '.'

describe('', () => {
  it('should render the colors correctly', () => {
    const { container } = render(<Main />)

    expect(container.firstChild).toHaveClass('bg-gray-50')
  })
})
