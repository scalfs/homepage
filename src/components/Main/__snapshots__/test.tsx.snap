// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` should render the heading 1`] = `
<main
  class="bg-[#06092b]"
>
  <h1>
    Vitor Escalfoni
  </h1>
  <h2>
    Software Engineer specialized in mobile app development.
  </h2>
</main>
`;
